src/controllers/institutes.js

```diff
@@ -1,8 +1,8 @@
-const { cloudinary } = require('../services/cloudinary');
+const { cloudinary } = require("../services/cloudinary");
 
-const Institute = require('../models/institute');
-const Event = require('../models/event');
-const User = require('../models/user');
+const Institute = require("../models/institute");
```

Comment:
🙄Since we are using airbnb style eslint strucutre for our api, its recommended to use single quotes for strings.